app/scripts/lib/channels/web.js

```diff
@@ -46,11 +46,6 @@ define([
         sender: sender,
         receiver: receiver
       });
-    },
-
-    teardown: function () {
-      this._sender.teardown();
-      this._receiver.teardown();
```

Comment:
Is this an unrelated cleanup?  It's fine, but seems out of place in this PR, so I wanted to sanity-check.